pkg/controller/garbagecollector/graph_builder.go

```diff
@@ -287,20 +296,22 @@ func (gb *GraphBuilder) IsSynced() bool {
 
 // Run sets the stop channel and starts monitor execution until stopCh is
 // closed. Any running monitors will be stopped before Run returns.
-func (gb *GraphBuilder) Run(stopCh <-chan struct{}) {
-	klog.Infof("GraphBuilder running")
-	defer klog.Infof("GraphBuilder stopping")
+func (gb *GraphBuilder) Run(ctx context.Context) {
+	logger := klog.FromContext(ctx)
+	logger = klog.LoggerWithValues(logger, "component", "GraphBuilder")
```

Comment:
*do* we want to add this to the context so that the downstream calls have it?

Reply:
Probably

Reply:
Let's add a name in app/core.go.

Reply:
No need to change anything regarding this in the PR, I'll cover it in a final cleanup PR that'll make all controllers consistent.